datadog-monitors-monitor-handler/src/main/java/com/datadog/monitors/monitor/ReadHandler.java

```diff
@@ -54,7 +54,10 @@
                 .build();
         }
 
+        final String monitorURL = model.getDatadogCredentials().getApiURL() + "/api/v1/monitor/" + model.getId().toString();
```

Comment:
Let's confirm with @softprops but I think they want the URL of the monitor to see in the UI right ?
which would be
```suggestion
        final String monitorURL = model.getDatadogCredentials().getApiURL() + "/monitors/" + model.getId().toString();
```

Reply:
so maybe we should call it `HTTP_URL`

Reply:
The reason for using my use of cfn is to abstract over the using the API directly.

The output of a monitor url in my case would be a feedback cycle to visually confirm that the generated monitor is what I expect.

Reply:
Would
```yaml
!Join
  - ''
  - - "https://app.datadoghq.com/monitors/"
    - !GetAtt "MyDDMonitor.Id"
```
work as a workaround for now?